### PR TITLE
Add missing VisitArbitrary methods in kubectl explain

### PR DESCRIPTION
**Pull request by soltysh**
**What type of PR is this?**
/kind bug
/priority important-longterm
/sig cli

**What this PR does / why we need it**:
This is alternative approach to https://github.com/kubernetes/kubernetes/pull/73817 and I think is a better one. 

**Which issue(s) this PR fixes**:
This was noticed in https://github.com/kubernetes/kubernetes/pull/71192

**Special notes for your reviewer**:
/assign @liggitt @sttts @roycaihw @apelisse 

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

### CODE REVIEW

File: pkg/kubectl/explain/field_lookup.go [hunk 1]
```diff
@@ -89,6 +89,12 @@ func (f *fieldLookup) VisitKind(k *proto.Kind) {
 	subSchema.Accept(f)
 }
 
+func (f *fieldLookup) VisitArbitrary(a *proto.Arbitrary) {
+	if f.SaveLeafSchema(a) {
```
Comment (apelisse) [inline pkg/kubectl/explain/field_lookup.go]: `f.Schema = a` would be more accurate (note that the condition is pointless).

**Comment by apelisse** [comment]
lgtm after nit, thanks @soltysh !

File: pkg/kubectl/explain/field_lookup.go (hunk 1 above)
Comment (soltysh) [reply pkg/kubectl/explain/field_lookup.go]: Make sense.

**Comment by soltysh** [comment]
Updated, @apelisse ptal

**Comment by apelisse** [comment]
Thank you!
/lgtm

**Pull request merged**